src/main/java/com/reckue/post/service/impl/NodeServiceImpl.java

```diff
@@ -142,145 +144,130 @@ public Node update(Node node, Map<String, Object> tokenInfo) {
-    }
-
-    /**
-     * This method is used to sort objects by id.
-     *
-     * @return list of objects of class Node sorted by id
-     */
-    public List<Node> findAllAndSortById() {
-        return findAll().stream()
-                .sorted(Comparator.comparing(Node::getId))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * This method is used to sort objects by status.
-     *
-     * @return list of objects of class Node sorted by status
-     */
-    public List<Node> findAllAndSortByStatus() {
-        return findAll().stream()
-                .sorted(Comparator.comparing(Node::getStatus))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * This method is used to sort objects by type.
-     *
-     * @return list of objects of class Node sorted by type
-     */
-    public List<Node> findAllAndSortByType() {
-        return findAll().stream()
-                .sorted(Comparator.comparing(Node::getType))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * This method is used to sort objects by userId.
-     *
-     * @return list of objects of class Node sorted by userId
-     */
-    public List<Node> findAllAndSortByUserId() {
-        return findAll().stream()
-                .sorted(Comparator.comparing(Node::getUserId))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * This method is used to sort objects by source.
-     *
-     * @return list of objects of class Node sorted by source
-     */
-    public List<Node> findAllAndSortBySource() {
-        return findAll().stream()
-                .sorted(Comparator.comparing(Node::getSource))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * This method is used to sort objects by createdDate.
-     *
-     * @return list of objects of class Node sorted by createdDate
-     */
-    public List<Node> findAllAndSortByCreatedDate() {
-        return findAll().stream()
-                .sorted(Comparator.comparing(Node::getCreatedDate))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * This method is used to get an object by id.
-     * Throws {@link NodeNotFoundException} in case if such object isn't contained in database.
-     *
-     * @param id object
-     * @return post object of class Node
-     */
-    @Override
-    public Node findById(String id) {
-        return nodeRepository.findById(id).orElseThrow(
-                () -> new NodeNotFoundException(id));
+        return dao.findAll(limit, offset, sort, desc);
     }
 
+//    /**
+//     * This method is used to sort objects in descending order by type.
+//     *
+//     * @param sort parameter for sorting
+//     * @param desc sorting descending
+//     * @return list of objects of class Node sorted by the selected parameter for sorting
+//     * in descending order
+//     */
+//    public List<Node> findAllByTypeAndDesc(String sort, boolean desc) {
```

Comment:
Don't collect garbage in the code. If you complite with this part, then remove it.